cmd/go-filecoin/address_daemon_test.go

```diff
@@ -1,86 +1,114 @@
 package commands_test
```

Comment:
Rename files to x_integration_test or x_cmd_test

Reply:
will do one big follow up